src/components/Theme/custom-media.css

```diff
@@ -1,6 +1,8 @@
 @custom-media --small-viewport (max-width: 479px);
 @custom-media --smaller-viewport (max-width: 359px);
 
+/* For viewport of shorter phones like iPhone SE (1st gen) */
+@custom-media --shorter-viewport (max-height: 480px);
```

Comment:
I found that on slightly bigger screens, the images scroll is still hidden by the upload button, e.g. in `520px` viewport height:

![2020-09-30_09-38-55](https://user-images.githubusercontent.com/1215541/94662875-e7df8180-0300-11eb-96e1-17370e98aedf.png)

```suggestion
@custom-media --shorter-viewport (max-height: 520px);
```

Reply:
I'll double check this breakpoint on a few mobile devices to make sure it it looks OK still, but I noticed there was definitely a difference in viewport dimensions between the browser device emulator vs actual device.